src/components/calcite-input-time-picker/calcite-input-time-picker.tsx

```diff
@@ -106,16 +106,22 @@ export class CalciteInputTimePicker {
   };
 
   private inputInputHandler = (event: CustomEvent): void => {
-    if (validateTimeString(event.detail.value) || !event.detail.value) {
-      this.value = event.detail.value;
+    const value = event.detail.value;
+    const validatedValue = validateTimeString(value);
+    if (validatedValue) {
+      debugger;
```

Comment:
Remove the debugger; (applies to other lines/files)

Reply:
Woopsie!